WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -857,15 +865,33 @@ extension NotificationsViewController {
                 self.notificationCommentDetailCoordinator.onSelectedNoteChange = { [weak self] note in
                     self?.selectRow(for: note)
                 }
-
                 commentDetailViewController.navigationItem.largeTitleDisplayMode = .never
-                self.showDetailViewController(commentDetailViewController, sender: nil)
 
+                if shouldPushDetailsViewController {
+                    commentDetailViewController.hidesBottomBarWhenPushed = true
+                    navigationController?.pushViewController(commentDetailViewController, animated: true)
+                } else {
+                    self.showDetailViewController(commentDetailViewController, sender: nil)
+                }
                 return
             }
+            if shouldPushDetailsViewController {
+                presentDetailsWithNoTabBar(for: note)
+            } else {
+                self.performSegue(withIdentifier: NotificationDetailsViewController.classNameWithoutNamespaces(), sender: note)
+            }
```

Comment:
[Suggestion] Can we get rid of the segue? We can create the same view controller programmatically for both cases and then, based on `shouldPushDetailsViewController`, decide whether to push it or use showDetail. And if we abstract this if condition to a helper function, as I mentioned in the other comment, we can use it here as well.